src/YoutubeAtom.tsx

```diff
@@ -209,70 +209,81 @@ export const YoutubeAtom = ({
         let hasSent50Event = false;
         let hasSent75Event = false;
 
-        const listener = player.current?.on(
```

Comment:
@liywjl these diffs look wrong to me - can you check if I'm removing code that I shouldn't be here?

Reply:
you are just removing optional chaining in this case. So its fine in my eyes

Reply:
But if you look at the larger diff, things seen very off, no?

Reply:
Double checked with @liywjl and these diffs are fine